### ISSUE TITLE
Updates to Scene Summary are ready

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#127]
@mbarlow12, based on feedback from screen reader consultants and our in-house content experts we have made some changes to the text on the opening scene/screen summary, the state of the sim list and the initial interaction hint.

**Goals of changes were to:**
- shorten things
- break it up into more manageable chunks
- make the ruler less prominent
- AND ensure force vectors and force are described correctly and together in a helpful way.

Please see the [start of the PDOM](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.jtxmm44qd6hc) in design document, I think I have marked all changes in red text indicates all the changes.

**Here's a snap shot:**
**Summary paragraph for single-screen sim**
{{Gravity Force Lab}} is an interactive sim. It changes as you play with it. It has a Play Area and Control Area. 

The Play Area has a blue sphere labeled m1 and a red sphere labeled m2. A force vector arrow starts at the center of each sphere and points directly at the opposite sphere. Spheres can be moved closer or further from one another. The mass of each sphere can be increased or decreased. Each sphere is held in place by a robot. 

In the Control Area, checkboxes change what things are shown and how they behave, and a button resets the sim. There is a moveable ruler to take measurements, if needed.

**State of sim in paragraph and list:** 
Currently, force on m1 by m2 is equal in magnitude and opposite in direction of force on m2 by m1.
- Force vector arrows are {{tiny}} {{at 0.041712 micronewtons}}.  
- m1 and m2 are {{not so close to}} each other, centers exactly {{4 meters apart}}.
- Mass of m1 is {{50}} kilograms, {{slightly smaller than}} m2 at {{200 kilograms}}.
- Robots {{pull very little}} keeping spheres in place.

**Sim specific interaction hint in its own paragraph:**
Change position and mass of spheres to explore.

**Keyboard shortcuts hint in its own paragraph (not changed):**
If needed, check out the Keyboard Shortcuts under Sim Resources.

**Comment by terracoda** [comment]
@mbarlow12, I highlighted the changed phrases to make them a little more obvious.

**Comment by terracoda** [comment]
@mbarlow12, please note the small change from "pulling" to "pull". I think that was a typo on my part in the design document (sorry).

**Comment by mbarlow12** [comment]
New dev versions for GFL (includes screen summary updates):
https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.10/phet/gravity-force-lab_en_phet.html?a11y
https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.10/phet/gravity-force-lab_a11y_view.html

Note: I somehow missed adding 'kilograms' to the interactive alerts for changing mass. Will be fixed in the next update.

**Comment by terracoda** [comment]
@mbarlow12, the scene/screen summary sounds great. In the a11y view, I noticed a missing period and a missing phrase. I'm not sure it is possible to add the missing phrase to the end of the first paragraph. It is possible that that phrase should be part of common code. Unfortunately, I do not know when I added that phrase.

If you can't put the missing phrase in the the first paragraph, please just add it to the beginning of the second.

**Here's an updated screen shot:**
<img width="459" alt="screen-summary" src="https://user-images.githubusercontent.com/1902116/50150435-48353880-02be-11e9-8391-9d87d47856ea.png">

**Comment by terracoda** [comment]
@mbarlow12, I think the missing phrase is actually part of common code.

**Comment by terracoda** [comment]
@mbarlow12, all the content changes looks good. We just need 2 small changes:
1. a period, at the end of the sentence "Spheres can be moved closer or further from one another.", and
2. change "labelled" to "labeled", unless there is a PhET practice to use British spelling?

Then we can close this issue.

**Comment by terracoda** [comment]
@zepumph, the period has been added.
Just need to to change "labelled" to "labeled" and we can close this issue.

@zepumph, this might be another easy one you can do that would benefit both sims, Regular and Basics.

Assigning to @zepumph.

**Comment by terracoda** [comment]
@zepumph, this should be an easy fix and we want "labeled" to be spelled the same across both sims, REGULAR and BASICS.

I am assuming it would be a good idea to address this issue before doing the wording simplifications for BASICS described in https://github.com/phetsims/gravity-force-lab-basics/issues/86

**Comment by zepumph** [comment]
usages of 'labelled' have been converted above. Please review @terracoda, anything else here?

**Comment by terracoda** [comment]
@zepumph! Verified. Looks good! Closing.
